addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/ICloudDataRequest.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal;
+
+/**
+ * Serializable request for icloud device data.
+ *
+ * @author Patrik Gfeller
+ *
+ */
+public class ICloudDataRequest {
```

Comment:
Does it work like this or does the GSON prefer constructors, adding those could be good way to get rid of those `SuppressWarnings` also.